.claude/skills/prow-ci/fetch_prow_artifacts.py

```
@@ -0,0 +1,217 @@
+def fetch_junit_xml(gcs_base_path, output_dir):
+    """Fetch JUnit XML from artifacts."""
+    gcs_path = f"{gcs_base_path}/artifacts/junit*.xml"
+
+    # List and download all JUnit files
+    try:
+        cmd = [
+            'gcloud', 'storage', 'ls',
+            f"{gcs_base_path}/artifacts/",
+            '--format=value(name)'
+        ]
+        result = subprocess.run(cmd, check=True, capture_output=True, text=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)